Plugins/DataSink/SQLite/SQLite.cs

```diff
@@ -107,8 +105,8 @@ private void InsertData(MeasurementValue sensorValue)
             string stm = $"INSERT INTO {sensorValue.Name} (timestamp, value, hash) VALUES (@timestamp, @value, @hash)";

```

Comment:
the unsanitized `sensorValue.Name` could potentionally be an attack vector for sql injection (this is not the only occurrence in this file, there are a few other statements with the same problem)

Reply:
Correct - I'm thinking about a different ERM. One table for measurement value, one table for units, one table for the source. Then all tablenames are fixed - this will make the implementation of the Web part more easier. And it will fix the injection problem.

Reply:
I'd be happy to take a look at it once you changed the ERM / have a concrete proposal :+1: